### PR TITLE
Removing the limitation on choice of plugin in Maya and Nuke

### PR DESCRIPTION
**Pull request by mkolar**
Currently it's impossible to submit other than Maya job from Maya and Nuke job from Nuke. This prevents user to use for instance arnold or vray standalone rendering with pyblish. 

I've fixed this by only specifying the deadline plugin if it's not already present in the data.

### CODE REVIEW

**Comment by mottosso** [comment]
Should probably gitignore the `us.stackdump` file.

**Comment by mkolar** [comment]
Ah yes. I missed that. Will fix

**Comment by mkolar** [comment]
Removed `us.stackdump`

File: pyblish_deadline/plugins/deadline.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def process(self, context):
             if instance.has_data('deadlineFrames'):
                 job_data['Frames'] = instance.data('deadlineFrames')
 
-            if not job_data['Plugin']:
+            if not job_data.get('Plugin'):
```
Comment (tokejepsen) [inline pyblish_deadline/plugins/deadline.py]: Shouldn't there be a default for the get method, something like ```job_data.get('Plugin', {})```?

Comment (mkolar) [inline pyblish_deadline/plugins/deadline.py]: I was always under the impression that get method on dictionary automatically returns None if not found and you use 'default' flag only if you want to return a different value.

Quite frankly I don't mind either of the solution that you suggested. As far as they all work (they should), I think it's splitting hairs.

Comment (tokejepsen) [inline pyblish_deadline/plugins/deadline.py]: You are right, didn't think about it returning ```None``` by default.

**Comment by tokejepsen** [comment]
Happy for @mkolar to merge.

**Pull request merged**